qiime2/plugin/plugin.py

```diff
@@ -170,23 +173,24 @@ def decorator(transformer):
             # Apply the decorator as we were applied with a single function
             return decorator(_fn)
 
-    def register_semantic_types(self, *semantic_types):
-        for semantic_type in semantic_types:
-            if not is_semantic_type(semantic_type):
-                raise TypeError("%r is not a semantic type." % semantic_type)
+    def register_semantic_type_fragments(self, *semantic_type_fragments):
```

Comment:
Please find and replace globally across the entire framework:

`s/register_semantic_types/register_semantic_type_fragments/g`

This rename is at least part of what is causing travis to fail at the moment. You should be able to verify by running the tests locally. :baggage_claim: